packages/account-v2/src/modules/src/POAForm/POAFormContainer.tsx

```diff
@@ -3,13 +3,13 @@ import { useHistory } from 'react-router-dom';
 import { useActiveTradingAccount } from '@deriv/api-v2';
 import { DerivLightApprovedPoaIcon, DerivLightDeclinedPoaIcon, DerivLightWaitingPoaIcon } from '@deriv/quill-icons';
 import { Button, Loader, Text } from '@deriv-com/ui';
-import { DemoMessage } from '../../components/DemoMessage';
-import { IconWithMessage } from '../../components/IconWithMessage';
-import { AUTH_STATUS_CODES } from '../../constants/constants';
-import { ACCOUNT_V2_ROUTES, P2P_ROUTE } from '../../constants/routes';
-import { AddressDetailsForm } from '../../containers/POAForm/AddressDetailsForm';
-import { usePOAInfo } from '../../hooks/usePOAInfo';
-import { isNavigationFromDerivGO, isNavigationFromP2P } from '../../utils/platform';
+import { DemoMessage } from '../../../components/DemoMessage';
+import { IconWithMessage } from '../../../components/IconWithMessage';
```

Comment:
**NIT**: Can we add an index file for all folders and export required code in the index?

Reply:
Updated.